pkgs/by-name/ji/jitsi-prosody-plugins/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, fetchFromGitHub
+, stdenvNoCC
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "jitsi-prosody-plugins";
+  version = "20240318";
+
+  src = fetchFromGitHub {
+    owner = "jitsi-contrib";
+    repo = "prosody-plugins";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-ZApx7Z08dr1EFL5eelFG3IrfrAOpg5JqUX6hbodGwuo=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out
+    rm LICENSE README.md */README*.md
+    rm -r images */docs
+    cp -r * $out/
+
+    runHook postInstall
+  '';
+
+  # nothing todo here
```

Comment:
typo: to do

(And it would be more useful to document _why_ - which is because it's all just Lua scripts.)